plugins/Denizen/scripts/mobs/modifiers/suffixes/exploding.dsc

```diff
@@ -0,0 +1,7 @@
+mob_mod_exploding:
```

Comment:
I feel like these modifiers are short enough to have `suffixes.dsc`/`prefixes.dsc` files, then make new files for bigger ones

Reply:
I was going off of [Adriftus'](https://github.com/Adriftus-Studios/network-script-data) system with this, but yeah you're right, will do.

- [ ] Consolidate small modifiers